retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/RetrofitMeterIdPrefixFunctionBuilder.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.retrofit2;
+
+import static java.util.Objects.requireNonNull;
+
+import javax.annotation.Nullable;
+
+/**
+ * Builds a {@link RetrofitMeterIdPrefixFunction}.
+ *
+ * @see RetrofitMeterIdPrefixFunction#builder(String)
+ */
+public final class RetrofitMeterIdPrefixFunctionBuilder {
+
+    private final String name;
+    @Nullable
+    private String serviceTagName;
+    @Nullable
+    private String defaultServiceName;
+
+    RetrofitMeterIdPrefixFunctionBuilder(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Make tag of {@link RetrofitMeterIdPrefixFunction} contains Retrofit service interface name.
```

Comment:
Should it be `Makes a tag ...`? (I think it's a typo previously.)

Reply:
Thanks! Rephrased the Javadoc to make it more comprehensive.